cluster.tf

```diff
@@ -88,6 +88,7 @@ resource "aws_eks_addon" "core" {
     "vpc-cni",
     "coredns",
     "aws-ebs-csi-driver",
+    "eks-node-monitoring-agent",
```

Comment:
I suspect they will want this to be conditional as the examples below since it's a new add-on.